core/src/test/java/io/aiven/klaw/service/TopicSyncControllerServiceTest.java

```diff
@@ -199,6 +402,79 @@ private List<Team> getAvailableTeams() {
   private void stubUserInfo() {
     when(handleDbRequests.getUsersInfo(anyString())).thenReturn(userInfo);
     when(userInfo.getTeamId()).thenReturn(101);
-    when(mailService.getUserName(any())).thenReturn("kwusera");
+    when(mailService.getUserName(any())).thenReturn(USERNAME);
+    when(commonUtilsService.getTenantId(eq(USERNAME))).thenReturn(TENANT_ID);
+  }
+
+  private SyncBackTopics createSyncBackTopic(String TypeOfSync, String[] topics) {
+    SyncBackTopics syncBackTopics = new SyncBackTopics();
+    syncBackTopics.setSourceEnv("1");
+    syncBackTopics.setTargetEnv("2");
+    syncBackTopics.setTypeOfSync(TypeOfSync);
+    syncBackTopics.setTopicIds(topics);
+    return syncBackTopics;
+  }
+
+  private Topic createTopic(Integer topicId, String topicName, String EnvId) {
+
+    Topic topic = new Topic();
+    topic.setTeamId(topicId);
+    topic.setTopicname(topicName);
+    topic.setEnvironment(EnvId);
+    topic.setNoOfPartitions(2);
+    topic.setNoOfReplcias("1");
+    topic.setTeamId(TENANT_ID);
+    return topic;
+  }
+
+  private ResponseEntity<ApiResponse> createAPIResponse(HttpStatus status, String resultStatus) {
+    return ResponseEntity.status(status)
+        .body(ApiResponse.builder().status(status).result(resultStatus).build());
+  }
+
+  private void MockMultipleTopics() {
+
+    when(handleDbRequests.requestForTopic(any()))
+        .thenReturn(
+            new HashMap<String, String>() {
+              {
+                put("result", "success");
+                put("topicId", "1");
+              }
+            })
+        .thenReturn(
+            new HashMap<String, String>() {
+              {
+                put("result", "success");
```

Comment:
```suggestion
                put("result", ApiResultStatus.SUCCESS.value);
```

Reply:
Have updated all "success" to ApiResultStatus.SUCCESS.value